File: app/src/main/java/cn/com/woong/readhub/resp/BaseResponse.java
package cn.com.woong.readhub.resp;

/**
 * BaseResponse
 */
public class BaseResponse {
    public String errorMsg;
    public boolean success;
    public int errorCode;
    public int code = -1;
    public long timestamp;
    public int status;

    public int pageSize;
    public int totalItems;
    public int totalPages;
}

File: app/src/main/java/cn/com/woong/readhub/network/resp/BaseResponse.kt
package cn.com.woong.readhub.network.resp

/**
 * BaseResponse
 */
open class BaseResponse {
    var errorMsg: String? = null
    var success: Boolean = false
    var errorCode: Int = 0
    var code = -1
    var timestamp: Long = 0
    var status: Int = 0

    var pageSize: Int = 0
    var totalItems: Int = 0
    var totalPages: Int = 0
}